samples/features/sql-big-data-cluster/deployment/private-bdc/scripts/deploy-private-aks-udr.sh

```diff
@@ -0,0 +1,136 @@
+export FWROUTE_NAME_INTERNET=bdcaksrouteinet
+
+
+# Create Azure Resource Group
+az group create -n $RESOURCE_GROUP -l $REGION_NAME
+ 
+# Create Azure Virtual Network to host your AKS cluster
+az network vnet create \
+    --resource-group $RESOURCE_GROUP \
+    --location $REGION_NAME \
+    --name $VNET_NAME \
+    --address-prefixes 10.0.0.0/8 \
+    --subnet-name $SUBNET_NAME \
+    --subnet-prefix 10.1.0.0/16
+ 
+
+SUBNET_ID=$(az network vnet subnet show \
+    --resource-group $RESOURCE_GROUP \
+    --vnet-name $VNET_NAME \
+    --name $SUBNET_NAME \
+    --query id -o tsv)
+
+
+# Add Azure firewall extension
+az extension add --name azure-firewall
+
+# Dedicated subnet for Azure Firewall (Firewall name cannot be changed)
+az network vnet subnet create \
+    --resource-group $RESOURCE_GROUP \
+    --vnet-name $VNET_NAME \
+    --name AzureFirewallSubnet \
+    --address-prefix 10.2.0.0/24
+
+# Create Azure firewall 
+az network firewall create -g $RESOURCE_GROUP -n $FWNAME -l $REGION_NAME --enable-dns-proxy true
+
+# Create public IP for Azure Firewall 
+az network public-ip create -g $RESOURCE_GROUP -n $FWPUBIP -l $REGION_NAME --sku "Standard"
+
+# Create IP configurations for Azure Firewall 
+az network firewall ip-config create -g $RESOURCE_GROUP -f $FWNAME -n $FWIPCONFIG_NAME --public-ip-address $FWPUBIP --vnet-name $VNET_NAME
+
+
+# Getting public and private IP addresses for Azure Firewall 
+export FWPUBLIC_IP=$(az network public-ip show -g $RESOURCE_GROUP -n $FWPUBIP --query "ipAddress" -o tsv)
+export FWPRIVATE_IP=$(az network firewall show -g $RESOURCE_GROUP -n $FWNAME --query "ipConfigurations[0].privateIpAddress" -o tsv)
+
+## Create an User defined route table
+az network route-table create -g $RESOURCE_GROUP --name $FWROUTE_TABLE_NAME
+
+# Create User defined routes 
+az network route-table route create -g $RESOURCE_GROUP --name $FWROUTE_NAME --route-table-name $FWROUTE_TABLE_NAME --address-prefix 0.0.0.0/0 --next-hop-type VirtualAppliance --next-hop-ip-address $FWPRIVATE_IP --subscription $SUBID
+
+az network route-table route create -g $RESOURCE_GROUP --name $FWROUTE_NAME_INTERNET --route-table-name $FWROUTE_TABLE_NAME --address-prefix $FWPUBLIC_IP/32 --next-hop-type Internet
+
+
+# Add FW Network Rules
+
+az network firewall network-rule create -g $RESOURCE_GROUP -f $FWNAME --collection-name 'aksfwnr' -n 'apiudp' --protocols 'UDP' --source-addresses '*' --destination-addresses "AzureCloud.$REGION_NAME" --destination-ports 1194 --action allow --priority 100
+az network firewall network-rule create -g $RESOURCE_GROUP -f $FWNAME --collection-name 'aksfwnr' -n 'apitcp' --protocols 'TCP' --source-addresses '*' --destination-addresses "AzureCloud.$REGION_NAME" --destination-ports 9000
+az network firewall network-rule create -g $RESOURCE_GROUP -f $FWNAME --collection-name 'aksfwnr' -n 'time' --protocols 'UDP' --source-addresses '*' --destination-fqdns 'ntp.ubuntu.com' --destination-ports 123
+
+# Add FW Application Rules
```

Comment:
Review the spacing after a comment throughout the scripts. In some cases you have an extra line, in others do not. Should be consistent - I suggest to remove the extra line.